CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to Altruism in tech
+
+Thank you for your interest in contributing to **Altruism in Tech** :tada:! 
+If you're interested in contributing, you should find all of the details here. We've centralized it into this repository, we'd like to keep things tidy and focused. Issues and roadmap goals will reflect changes across multiple repositories on Github, so you'll have the opportunity to contribute to multiple aspects of this project.
+
+**Altruism in tech** is promoting diversity and inclusion in tech by connecting newcomers with tech professionals.
+
+This document is a set of guidelines for contributing to **Altruism in tech** on GitHub. These are guidelines, not rules. This guide is meant to make it easy for you to get involved.
+
+* [Participation guidelines](#participation-guidelines)
+* [What we're working on](#what-were-working-on)
+* [How to submit changes](#how-to-submit-changes)
+* [How to report bugs](#how-to-report-bugs)
+* [Communication channels](#communication-channels)
+
+## Participation guidelines
+
+This project adheres to a [code of conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to ipsha21@gmail.com.
+
+## What we're working on
+
+Take a look at this [Roadmapping exercise](https://github.com/ipsha21/altruism-in-tech/issues) to start making your own milestones and issues. 
```

Comment:
We could probably just limit this section to: "Take a look at the issues in our [current milestone](https://github.com/ipsha21/altruism-in-tech/issues) or browse for some good first bugs to get started!", since the roadmapping exercise is still under construction. :) 
I'll link it here once it's done.